src/main/java/core/basesyntax/impl/AmountFromFileImpl.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.db.FruitShopDao;
+import core.basesyntax.service.FruitTransaction;
+import core.basesyntax.service.AmountFromFile;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AmountFromFileImpl implements AmountFromFile {
+    private FruitShopDao fruitShopDao;
```

Comment:
The field `fruitShopDao` should be private and final to ensure immutability where possible, unless there's a need to change its reference after object construction.